gradlew.ps1

```diff
@@ -0,0 +1,80 @@
+# ##########################################################################
+#
+# Gradle startup script for Windows Powershell.
+#
+# Ported from gradlew.bat for use on Windows where .bat files are blocked.
+#
+# Can be use from Powershell Terminal in Visual Studio Code.
+#  Example usage to build :
+#    .\gradlew.ps1 build
+#
+# TODO : Does it help to install powershell extensions plugin in VSCode??
+#
+# Original gradlew.bat file is licensed under the Apache license.
+################################################################################
+#
+# Copyright 2015 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ##########################################################################
+
+$APP_BASE_NAME = $pwd
```

Comment:
In the original script this is also called APP_HOME, which I'd recommend using instead of APP_BASE_NAME (which is now what APP_NAME below represents).  That makes it more clear this is the "home" directory of the app.

Reply:
Fixed.